### ISSUE TITLE
Error while extracting (Unsupported compression method)

### ISSUE THREAD

**Issue by KParthSingh** [M2Team/NanaZip#390]
Yes I tried to extract the file in 7zip and this problem is nanazip specific. 7zip does not have this issue

How to reproduce:
Download this [WSA Build file](https://github.com/MustardChef/WSABuilds/releases/download/Windows_11_2311.40000.5.0/WSA_2311.40000.5.0_x64_Release-Nightly-MindTheGapps-13.0-RemovedAmazon.7z) while unzipping it show these following errors

```
D:\Softwares\WSA_2311.40000.5.0_x64_Release-Nightly-MindTheGapps-13.0-RemovedAmazon.7z
Unsupported compression method : WSA_2311.40000.5.0_x64\WSACrashUploader\arm64\KernelTraceControl.dll
Unsupported compression method : WSA_2311.40000.5.0_x64\WSACrashUploader\arm64\msdia140.dll
Unsupported compression method : WSA_2311.40000.5.0_x64\WSACrashUploader\arm64\msvcp140.dll
Unsupported compression method : WSA_2311.40000.5.0_x64\WSACrashUploader\arm64\vcruntime140.dll
```
As I mentioned earlier these error only show up in nanazip. not in 7zip or WinRAR.

**Comment by MouriNaruto** [comment]
I have reproduced the issue and need to do some researches.

Kenji Mouri

**Comment by MouriNaruto** [comment]
I tried the NanaZip Preview (3.0) and it looks OK for this archive file.

I will try my best to develop the next release and release it as soon as possible.

Kenji Mouri

**Comment by KParthSingh** [comment]
> I tried the NanaZip Preview (3.0) and it looks OK for this archive file.

Sorry my bad. I was using version 2.0 but it is working properly in preview 3.0 :)

**Comment by MouriNaruto** [comment]
This issue won't be closed until 3.0 is released.

Kenji Mouri

**Comment by MouriNaruto** [comment]
The NanaZip 3.0 Update 1 had been released for a long time.

Kenji Mouri
